omaha_server/omaha/request.xsd

```diff
@@ -90,7 +91,10 @@
             <xs:attribute name='experiments' use='optional' type="xs:string"/>
             <xs:attribute name='iid' use='optional' type="GuidType"/>
             <xs:attribute name='installage' use='optional' type='xs:integer'/>
+            <xs:attribute name='installdate' use='optional' type='xs:integer'/>
             <xs:attribute name='tag' use='optional' type='xs:string'/>
+            <xs:attribute name='_numaccounts' use='optional' type='xs:string'/>
```

Comment:
BTW, this fields is software-specific fields (in this case, chromium-specific), and can be removed.